app/models/rejection.rb

```diff
@@ -1,6 +1,7 @@
 class Rejection < ActiveRecord::Base
   REASONS = %w[
     slot_unavailable
+    no_adult
```

Comment:
@tatyree Yes, we will. We should re-evaluate metrics at some point, I agree, but that won't be for a while. In the meantime please add the column.